lib/types/skip-reason.ts

```diff
@@ -34,6 +32,7 @@ export type SkipReason =
   | 'unknown-registry'
   | 'unknown-version'
```

Comment:
```suggestion
```

Reply:
Is it okay to replace all instances of "unknown-version" with "unspecified-version"? 
"unknown-version" was found in 14 files with a total of 57 occurrences.